frontend/src/pages/SettingsRouter/SettingsRouter.tsx

```diff
@@ -117,6 +143,13 @@ export const SettingsRouter = () => {
 		],
 	]
 
+	const projectSettingTabs = allProjects
```

Comment:
**non-blocking**

These sorts of functions can be wrapped with a `useMemo`. The primary reason for `useMemo` is to maintain stable references. The secondary reason is to cut down on expensive operations.

This particular operation isn't particularly expensive, so it's up to you.